content/articles/data-entry-support-with-html/index.md

```diff
@@ -0,0 +1,123 @@
+### Data Entry Support with HTML
+
```

Comment:
Your introduction, key takeaways, and prerequisites should come before the table of contents.
arrange them in this order:
- Introduction
- Key takeaways
- Prerequisites
- Table of contents